src/fairseq2/datasets/speech_parquet.py

```diff
@@ -0,0 +1,319 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Same thinking as above with the asr_parquet.py dataset.

Who is using this, what dataset and is it tied to the open-source recipes? Otherwise it does not make sense to include in fairseq2.